### PR TITLE
ADGetGroupMembers empty field fix

### PR DESCRIPTION
**Pull request by shaniacht1**
Script would break if memberOf field in answer was empty.

### CODE REVIEW

File: Scripts/script-ADGetGroupMembers.yml [hunk 1]
```diff
@@ -10,7 +11,7 @@ script: |+
           endpoint['Type'] = 'AD'
           endpoint['ID'] = demisto.get(l,'dn')
           endpoint['Hostname'] = demisto.get(l,'name')
-          endpoint['Groups'] = demisto.get(l,'memberOf').split('<br>')
+          endpoint['Groups'] = demisto.get(l,'memberOf').split('<br>') if demisto.get(l,'memberOf') else demisto.get(l,'memberOf')
```
Comment (asafshen) [inline Scripts/script-ADGetGroupMembers.yml]: whats the point of assigning `Groups` fields in `demisto.get(l,'memberOf')` if its falsely?
I recommend:
1. assign `None` to it
OR
2. Don't assign this field at all:
```
if demisto.get(l,'memberOf'):
    endpoint['Groups'] = demisto.get(l,'memberOf').split('<br>')
```

Comment (dorsha) [reply Scripts/script-ADGetGroupMembers.yml]: Also, the one liner syntax is very hard to understand

File: Scripts/script-ADGetGroupMembers.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ commonfields:
   id: ADGetGroupMembers
   version: -1
 name: ADGetGroupMembers
+releaseNotes: "fixed script failing on empty fields"
```
Comment (asafshen) [inline Scripts/script-ADGetGroupMembers.yml]: please change to `Fixed script failure when user/computer has no groups`

File: Scripts/script-ADGetGroupMembers.yml (hunk 1 above)
Comment (orius123) [reply Scripts/script-ADGetGroupMembers.yml]: I think it's readable (this is python ternary operation)
`demisto.get(l,'memberOf').split('<br>') if demisto.get(l,'memberOf') else ''` or `else None`
or
`demisto.get(l,'memberOf') and demisto.get(l,'memberOf').split('<br>')`

Comment (dorsha) [reply Scripts/script-ADGetGroupMembers.yml]: I know it is python ternary.. but it is still hard to understand.. you can write everything in one liner..

**Pull request merged**